package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "react-gm-2020",
+  "version": "1.0.0",
+  "description": "React global mentoring program",
+  "main": "index.js",
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server --config ./config/webpack.dev.js",
+    "test": "jest --passWithNoTests",
+    "build:dev": "cross-env NODE_ENV=development webpack --config config/webpack.dev.js",
+    "build:prod": "cross-env NODE_ENV=production webpack --config config/webpack.prod.js"
+  },
+  "dependencies": {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-redux": "^7.2.1",
+    "react-router": "^5.2.0",
+    "redux": "^4.0.5"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.10.5",
+    "@babel/core": "^7.10.5",
+    "@babel/preset-env": "^7.10.4",
+    "@babel/preset-react": "^7.10.4",
+    "babel-jest": "^26.2.2",
+    "babel-loader": "^8.1.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "cross-env": "^7.0.2",
+    "css-loader": "^4.2.0",
+    "html-webpack-plugin": "^4.3.0",
+    "jest": "^26.1.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "react-test-renderer": "^16.13.1",
```

Comment:
`react-test-renderer` isn't the best tool for testing react components. Better use [enzyme](https://enzymejs.github.io/enzyme/) which built on the top of react test utils and have a rich API for manipulating components.
Or [testing-library](https://testing-library.com/), which is my favorite, because it renders component into DOM and provides a lot of handy helpers like getting dom node by text/aria-label/etc or async